### DISCUSSION TITLE
Switch states incorrect on container start

### DISCUSSION THREAD

**Discussion by PeteRager** [zwave-js/zwave-js#4406] [General] [answered]
I’m having this same issue with multiple device, device types and vendors.  Here’s a use case with a switch.

1. container is running, switch is on and jwavejs reports switch on - this is good!
2. Stop container
3. Turn switch off
5. Start container
6. Zwavejs reports switch is on - this is bad
7.  Wait for while - still reporting incorrectly. 
8.  Refresh the switch in control panel and it’s reported correctly.  Or physically change the switch state. 

Similar issue with thermostats.  On startup they report the last state - like heating with wrong setpoint. 

Looking at the logs, it appear that on startup the dynamic entities (like switch state) are not queried and a cached value is being used - resulting in incorrect data.

**Comment by AlCalzone** [comment, marked as answer]
This is intended and has been like this since v7.0.0
https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG_v8.md#700-2021-03-23--summer-is-coming

Querying a lot of potentially stale values during the startup lead to network congestion and delays. In larger network, these queries could take up to several minutes, during which normal communication could be negatively impacted. If you need some values to be up to date, you can query them yourself via an automation for example.

**Reply by PeteRager** [reply]
Makes sense.  I only see the issue when I take the zwavejsmqtt container down and start back up.  So yes, I can run an automation when the node status changes to available.  

It’d be nice to have a “poll once at startup” capability.
